bin/Powheg/make_rwl.py

```diff
@@ -20,47 +20,51 @@
 
 for m_rensc in m_factor :
   for m_facsc in m_factor :
-    fout.write("<weight id='"+str(m_idx)+"'> lhapdf=260000 renscfact="+ \
+    fout.write("<weight id='"+str(m_idx)+"'> lhapdf=305800 renscfact="+ \
```

Comment:
I believe we want to go to NNLO for the central value, there is a mistake in the google spreadsheet, therefore 306000 and similar